generator/C/include_v2.0/mavlink_helpers.h

```diff
@@ -708,12 +704,13 @@ MAVLINK_HELPER uint8_t mavlink_frame_char_buffer(mavlink_message_t* rxmsg,
 			status->parse_state = MAVLINK_PARSE_STATE_GOT_PAYLOAD;
 		}
 #ifdef MAVLINK_CHECK_MESSAGE_LENGTH
-	        if (rxmsg->len != MAVLINK_MESSAGE_LENGTH(rxmsg->msgid))
-		{
+        if (rxmsg->len < mavlink_min_message_length(rxmsg) ||
+            rxmsg->len > mavlink_max_message_length(rxmsg))
+        {
```

Comment:
Not that I like it, but the code around here has tabs as the indenting, you've put spaces here (and below), which makes it look bad.

Reply:
I agree. There are spaces and tabs in this file, it would be nice to decide on one and fix it.

Reply:
In ArduPilot we use spaces so I'm sure that's the preference on our side. If PX4 is the same, then I would say that a commit uniforming it to spaces would be well accepted.

Reply:
So here are some stats for the status quo for all files in generator/C/include_v2.0/:

4 spaces: 1305 matches
tab: 1295 matches

It's a close race! :smile:

Reply:
I'm fine to make a PR to make it consistent but it should probably be a separate PR.